docs/sources/explore/metrics-explore.md

```diff
@@ -21,51 +21,46 @@ Metric Explore is currently in [private preview](/docs/release-life-cycle/). Gra
 With Metrics Explore, you can:
 
 - easily slice and dice metrics based on their labels, so you can immediately see anomalies and identify issues 
-- see the right visualization for your metric based on its type (gauge vs. counter, for example) without writing it yourself
+- see the right visualization for your metric based on its type (gauge vs. counter, for example) without building it yourself
 - surface other metrics relevant to the current metric
 - “explore in a drawer” - expand a drawer over a dashboard with more content so you don’t lose your place
 - view a history of user steps when navigating through metrics and their filters
 - easily pivot to other related telemetry, including logs or traces 
 
-You can do all of these things without writing a single query!
-
-You can access Metrics Explore either as a standalone experience or as part of Grafana Dashboards. 
+You can access Metrics Explore either as a standalone experience or as part of Grafana dashboards. 
 
 ## Standalone experience
 
 To access Metrics Explore as a standalone experience:
 
-1. Click the arrow next to **Explore** in the Grafana left-side menu and click **Metrics**. You will be taken to an overview page that shows recent metrics, bookmarks, and the option to select a new metric exploration.
-1. Click **+ New metric exploration**.
-1. Select **Prometheus** in the drop-down menu under **Data source**. You can also select a Prometheus-compatible data source available in the list.
-1. Click **+ Add label** to select a label from the drop-down menu. You can add multiple labels.  A label type will appear above the label with a drop-down of options. For example, if you select the label `container` a drop-down list of available containers appears.
+1. Click the arrow next to **Explore** in the Grafana left-side menu and click **Metrics**. You are taken to an overview page that shows recent metrics, bookmarks, and the option to select a new metric exploration. 
+1. To get started with a new exploration, click **+ New metric exploration**.
+1. Select **Prometheus** in the drop-down menu under **Data source**. You can also select any Prometheus-compatible data source available in the list.
+1. Click **+ Add label** to select a label from the drop-down menu. You can add multiple labels. A label type will appear above the selected label with a drop-down list of options from which to choose. For example, if you select the label `container` a drop-down list of available containers appears.
```

Comment:
I would say: Click "Add Label" to select a label-value pair from the drop-down menu. You can add multiple label-value pairs. You can add multiple label-value pairs. The value will appear above the selected label ...."

Reply:
done!